mgmt/Alarms.cc

```diff
@@ -135,15 +133,17 @@ Alarms::resolveAlarm(alarm_t a, char *ip)
     snprintf(buf, sizeof(buf), "%d-%s", a, ip);
   }
 
-  if (!ip && ink_hash_table_lookup(local_alarms, buf, &hash_value) != 0) {
-    ink_hash_table_delete(local_alarms, buf);
-    ats_free(((Alarm *)hash_value)->description);
+  if (!ip && local_alarms.find(buf) != local_alarms.end()) {
+    Alarm *hash_value = local_alarms[buf];
+    local_alarms.erase(buf);
+    ats_free(hash_value->description);
     ats_free(hash_value);
-  } else if (ip && ink_hash_table_lookup(remote_alarms, buf, &hash_value) != 0) {
+  } else if (ip && remote_alarms.find(buf) != remote_alarms.end()) {
     char buf2[1024];
```

Comment:
I realize this is outside the scope of the PR, but what the heck is the point of `buf2`?

Reply:
Right. It makes no sense. Maybe someone forgot to remove it when they made changes here.